controllers/article-controllers.js

```diff
@@ -19,7 +20,22 @@ exports.getArticleById = (req, res, next) => {
 };
 
 exports.getAllArticles = (req, res, next) => {
-  returnArticles(req.query)
+  const { topic } = req.query;
+
+  getAllTopics()
+    .then((topics) => {
+      const foundTopic = topics.find((topicObj) => topicObj.slug === topic);
+      if (topic === undefined) {
+        return undefined;
```

Comment:
This works, but we could potentially tidy this up by using a utils function to query the topics table and check that the passed topic exists. Up to you if you'd like to refactor at some point 🙂